tests/integration_tests/fixtures/importexport.py

```diff
@@ -496,6 +497,7 @@
     "params": None,
     "template_params": {},
     "normalize_columns": False,
+    "time_secondary_column": False,
```

Comment:
We need to make sure that we can still import exports that have `time_secondary_column` set.

Reply:
Datasets exports and imports successfully with the `time_secondary_column`.